cwel/component/button/button.es

```diff
@@ -0,0 +1,15 @@
+angular.module('cwel').directive('hoverEffect', () => {
+    return {
+        restrict: 'A',
```

Comment:
@NathanRodger I would restrict this to 'C' as this behaviour is unique to buttons at the moment.